open-metadata-implementation/repository-services/repository-services-implementation/src/main/java/org/odpi/openmetadata/repositoryservices/localrepository/repositoryconnector/LocalOMRSMetadataCollection.java

```diff
@@ -5368,19 +5368,11 @@ public Relationship restoreRelationship(String    userId,
             throw new UserNotAuthorizedException(error);
         }
 
-        /*
-         * Check relationship is valid
-         */
-        Relationship relationship = realMetadataCollection.getRelationship(userId, deletedRelationshipGUID);
-        repositoryValidator.validateRelationshipFromStore(repositoryName, deletedRelationshipGUID, relationship, methodName);
-
         /*
          * Restore relationship
          */
         Relationship   newRelationship = realMetadataCollection.restoreRelationship(userId, deletedRelationshipGUID);
 
-        repositoryValidator.validateRelationshipFromStore(repositoryName, deletedRelationshipGUID, newRelationship, methodName);
```

Comment:
This one should be ok - we are validating the relationship is not deleted after it has been restored.

Reply:
Apologies for merging this quickly, I am trying to get it in master and include it as part of the lineage testing. Can we fix it forward with new PR putting it back?